stan/math/prim/fun/sd.hpp

```diff
@@ -23,10 +23,10 @@ namespace math {
  */
 template <typename T, require_container_t<T>* = nullptr,
           require_not_st_var<T>* = nullptr>
-inline auto sd(const T& m) {
+inline auto sd(T&& m) {
   using std::sqrt;
 
-  return apply_vector_unary<T>::reduce(m, [](const auto& x) {
+  return apply_vector_unary<T>::reduce(std::forward<T>(m), [](const auto& x) {
```

Comment:
```suggestion
  return apply_vector_unary<T>::reduce(std::forward<T>(m), [](auto&& x) {
```

Then can forward to `variance()` below

Reply:
👍